types/heapdump/index.d.ts

```diff
@@ -3,4 +3,4 @@
 // Definitions by: weekens <https://github.com/weekens>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export function writeSnapshot(dumpFileName: string, callback: (err?: Error) => void): void;
+export function writeSnapshot(dumpFileName?: string, callback?: (err?: Error, filename?: string) => void): void;
```

Comment:
If the callback *always* returns the name of the file, this should be `(err: Error | undefined, filename: string) => void`.

Reply:
Good rectification